specifications/api/swagger.yaml

```
@@ -2114,18 +2114,36 @@ components:
         - alonzo
         - babbage
 
+    NetworkId: &NetworkId
+      type: object
+      required:
+        - magic
+        - id
+      properties:
+        magic:
+          description: The unique magic number defining the network the wallet is working on.
+          type: integer
+        id:
+          description: A name just distinguishing mainnet from testnets
+          type: string
+          enum:
+              - mainnet
+              - testnet
+
     ApiNetworkInformation: &ApiNetworkInformation
       type: object
       required:
         - sync_progress
         - node_tip
         - node_era
+        - network_id
```

```suggestion
        - network_info
```

The API currently shows:
```
$ curl -X GET http://localhost:8090/v2/network/information | jq

{
  "network_id": {
    "magicNumber": 1097911063,
    "networkId": "testnet"
  },
....
```
In general we are using `snake_case` rather then `camelCase` in JSON returned by the API so maybe for consistency it'd be better to change to something like:

```
  "network_info": {
    "magic_number": 1097911063,
    "network_id": "testnet"
  },
```